app/controllers/organisations_controller.rb

```diff
@@ -1,38 +1,50 @@
 class OrganisationsController < ApplicationController
   before_action :set_organisation, except: [:index, :new, :create]
+  after_action :verify_authorized, except: :index
+  after_action :verify_policy_scoped, only: :index
 
   def index
-    @organisations = Organisation.all
+    @organisations = policy_scope(Organisation)
   end
 
   def show
+    authorize @organisation
   end
 
   def new
     @organisation = Organisation.new
+    authorize @organisation
   end
 
   def create
     @organisation = Organisation.new(organisation_params)
+
+    authorize @organisation
+
     if @organisation.save
-      redirect_to organisations_path, notice: flash_message(:create, Organisation)
+      redirect_to organisation_path(@organisation), notice: flash_message(:create, Organisation)
     else
       render :new
     end
   end
 
   def edit
+    authorize @organisation
   end
 
   def update
+    authorize @organisation
```

Comment:
I think the `authorize` should be done after the updated attributes are assigned, so that we verify the final state.